dashboard/lib/services/curriculum_pdfs/resources.rb

```diff
@@ -98,8 +98,9 @@ def generate_script_resources_pdf(script, directory = "/tmp/")
               "temporary directory contents: #{Dir.entries(pdfs_dir).inspect}",
               color: 'red'
             )
+            # Group ID: S07FB3XSAR5 - teacher-tools-on-call would be notified
             ChatClient.log(
-              "<@teacher-tools-on-call> Please follow instructions in https://docs.google.com/document/d/1mBY56DeAzrwTM3CVIOFho3azTi9mudE37ZQrVZXxaMA/edit#heading=h.axfu5or8cueg to troubleshoot",
+              "<!subteam^S07FB3XSAR5> Please follow instructions in https://docs.google.com/document/d/1mBY56DeAzrwTM3CVIOFho3azTi9mudE37ZQrVZXxaMA/edit#heading=h.axfu5or8cueg to troubleshoot",
```

Comment:
This just feels really unreadable... I was really expecting this to look how you originally had it (and how it is shown [here)](https://github.com/code-dot-org/code-dot-org/blob/40cdbf241213f1d74d8e965f2998855cee4e1d09/bin/cron/check_invalid_staging_filenames#L16)?

Reply:
After looking at `chat_client`, I don't really have a better idea here, but I don't think this is how this was initially imagined on working.  I suppose we can use it for now to test it out.  Perhaps tag Darin to see if he has insight on how he imagined it being used.

Reply:
Yup, that's what I was also expecting, but their documentation suggests using group ID for group notification vs user names for individual users. Another option is to tag DOTD and have them engage the teacher tools on call, but that seems a bit unnecessary. 

I like merging this to first validate the changes (I'm hesitant to use slack bot token in my locals.yml) so I'll validate it and continue following up on the right implementation.

Reply:
After some more thinking, I changed my mind and wanted to clean up before merging. The latest iteration includes some changes to chat client to enable this scenario in a more readable way. I'll involve infra in this review as well.